firefox-ios/EcosiaTests/Account/Auth/AuthTests.swift

```diff
@@ -444,4 +529,11 @@ final class AuthTests: XCTestCase {
         mockProvider.startAuthCallCount = 0
         mockProvider.storeCredentialsCallCount = 0
     }
+
+    private func setupLoggedInStateWithSSOProvider() async {
+        // For unit testing, we use the regular mock provider
+        // The SSO functionality requires integration testing or a more sophisticated mocking approach
+        // that can bypass the type check in Auth.retrieveSSOCredentials()
+        await setupLoggedInState()
+    }
```

Comment:
I don't get why this is needed, isn't it the same as just using `await setupLoggedInState()` as some tests already do?